glyphs-reader/src/font.rs

```diff
@@ -410,7 +304,7 @@ impl TryFrom<BTreeMap<String, Plist>> for Component {
         let mut transform = if let Some(plist) = dict.remove("transform") {
             Affine::from_plist(plist)
         } else {
-            Affine::identity()
+            Default::default()
```

Comment:
Purely personal preference, but as a reader, `Affine::IDENTITY` here would provide some clarity.

Reply:
Ha, apparently I'm looking at a version of the kurbo docs prior that existing. Fixed.